finetune_adapter.py

```diff
@@ -183,7 +186,11 @@ def pad_right(x, pad_id):
 
     x = torch.stack([pad_right(x, pad_id=0) for x in input_ids])
     y = torch.stack([pad_right(x, pad_id=-1) for x in labels])
-    x, y = fabric.to_device((x.pin_memory(), y.pin_memory()))
+
+    if fabric._accelerator.__class__.__name__ == "MPSAccelerator":
```

Comment:
This can be an isinstance check

Reply:
Updated with this [commit](https://github.com/Lightning-AI/lit-parrot/pull/94/commits/37264f798a0b4cdc93414d74daaabb56991a313d)